isis/src/qisis/objs/ControlPointEditWidget/ControlPointEditWidget.cpp

```diff
@@ -638,6 +701,38 @@ namespace Isis {
   }
 
 
+  /**
+   * Load ground measure into right side and add to file combo boxes.
+   *
+   * @author 2013-12-06 Tracie Sucharski
+   *
+   * @internal
+   *   @history 2013-12-06 Tracie Sucharski - Original version.
+   *   @history 2015-05-19 Ian Humphrey and Makayla Shepherd - moved duplicated code to
+   *                           findPointLocation() and createTemporaryGroundMeasure().
+   *
+   */
+  void ControlPointEditWidget::loadGroundMeasure() {
```

Comment:
is this another "measure" vs ground point thing?

Reply:
This is not, it is a ControlMeasure that is within a Fixed or Constrained ControlPoint.